pages/reporting/rops/index.js

```diff
@@ -0,0 +1,38 @@
+const { Router } = require('express');
+const routes = require('./routes');
+const { NotFoundError } = require('../../../lib/errors');
+
+module.exports = settings => {
+  const app = Router();
+
+  app.get('/', (req, res, next) => {
+    const year = new Date().getFullYear();
+    res.redirect(req.buildRoute('reporting.rops.summary', { year }));
+  });
+
+  app.param('year', (req, res, next, year) => {
+    req.year = parseInt(year, 10);
+    res.locals.static.year = req.year;
+    next();
+  });
+
+  app.get('/:year', (req, res, next) => {
+    if (!req.user.profile.asruRops) {
```

Comment:
As above. Can we base this on calculated permissions rather than spreading the logic around?